src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,14 +1,52 @@
 package core.basesyntax.service;
 
 import core.basesyntax.dao.StorageDao;
-import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.InvalidInputDataException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
-    private final StorageDao storageDao = new StorageDaoImpl();
+    private static final int MIN_AGE = 18;
+    private static final int MIN_AMOUNT_OF_CHARACTERS = 6;
+    private StorageDao storageDao;
+
+    public RegistrationServiceImpl() {
```

Comment:
Remove public RegistrationServiceImpl() and public RegistrationServiceImpl(StorageDao storageDao)